sendgrid.go

```diff
@@ -125,3 +125,12 @@ func MakeRequestAsync(request rest.Request) (chan *rest.Response, chan error) {
 
 	return r, e
 }
+
+// CheckSecrets checks email content for presence of an API key before sending.
+func CheckSecrets(mailContent string) (int, error) {
```

Comment:
I would suggest returning only a bool instead of an int and an error. And consider renaming `CheckSecrets` to `HasSecrets` or `HasAPIKey`.

Something like:
```golang
func HasAPIKey(mailContent string) bool {
   return apiKeyRegex.MatchString(mailContent)
}
```